### ISSUE TITLE
Frontend compile warning about DoubleScrollbar

### ISSUE THREAD

**Issue by imnasnainaec** [sillsdev/TheCombine#2020]
**Describe the bug**

Compiling the frontend generates the following warning:
```
Compiled with warnings.

Failed to parse source map from '/home/runner/work/TheCombine/TheCombine/node_modules/react-double-scrollbar/dist/DoubleScrollbar.js.map' file: Error: ENOENT: no such file or directory, open '/home/runner/work/TheCombine/TheCombine/node_modules/react-double-scrollbar/dist/DoubleScrollbar.js.map'
```

This is an upstream issue: https://github.com/material-table-core/core/issues/467

**To Reproduce**

`npm run build` or `npm run frontend`

Or see it in the GitHub actions, e.g.:
https://github.com/sillsdev/TheCombine/actions/runs/4657985804/jobs/8243137786#step:11:10
https://github.com/sillsdev/TheCombine/actions/runs/4657985804/jobs/8243137050#step:5:205

**Comment by imnasnainaec** [comment]
Submitted fix for the upstream issue: https://github.com/material-table-core/core/pull/804

**Comment by imnasnainaec** [comment]
Fix has been approved and merged into their `master` branch.

**Comment by imnasnainaec** [comment]
They released [v6.3.0](https://github.com/material-table-core/core/releases/tag/v6.3.0).
